flight/PiOS/STM32F30x/library.mk

```diff
@@ -12,6 +12,7 @@ CDEFS				+= -DSTM32F30X
 CDEFS				+= -DHSE_VALUE=$(OSCILLATOR_FREQ)
 CDEFS 				+= -DUSE_STDPERIPH_DRIVER
 ARCHFLAGS			+= -mcpu=cortex-m4 -march=armv7e-m -mfpu=fpv4-sp-d16 -mfloat-abi=hard
+FLOATABI			= hard
```

Comment:
What is this doing? Is that relevant to changing the UAVOs? The git commit message doesn't mention this, but it looks like it's specifying to use the hardware FPU?